### ISSUE TITLE
Compatibility vue-cli-5

### ISSUE THREAD

**Issue by Maqsyo** [m0dch3n/vue-cli-plugin-cordova#145]
Please add compatibility with the new vue-cli-5 and webpack 5.
The serve process is working pretty fine, but i couldn't build with it.

**Comment by andrii-trush** [comment]
@Maqsyo did you solve this issue?

**Comment by Maqsyo** [comment]
@andrii-trush nope. I couldn't find any solution with enabled legacy mode. But the most mobile phones doesn't require this mode anymore. So I just kicked out the incompatible android/iOS Versions

**Comment by ifredom** [comment]
最终我只能降级 @vue-cli

`npm install -g @vue/cli@4`

**Comment by ifredom** [comment]
The final resolution version is to downgrade @vue-cli

`npm install -g @vue/cli@4`

**Comment by Maqsyo** [comment]
It shouldn't

But we upgraded our apps anyway to vite, so it isn't relevant for us anymore, but still important for other legacy projects or users
